website_form_builder/static/src/js/snippets.js

```diff
@@ -26,27 +25,27 @@ odoo.define('website_form_builder.snippets', function (require) {
     /**
      * Lazily ask just once for models.
      *
+     * @param {Class} servicesMixin
```

Comment:
In JS there are no classes; they are `Object`s.

```suggestion
     * @param {Object} servicesMixin
```